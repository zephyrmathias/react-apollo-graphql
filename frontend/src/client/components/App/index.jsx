import React from 'react';
import { hot } from 'react-hot-loader';
import Header from '../Header';
import AppRoutes from '../../routes/AppRoutes';

const App = () => (
  <div>
    <Header />
    <div>
      App
    </div>
    <AppRoutes />
  </div>
);

export default hot(module)(App);
