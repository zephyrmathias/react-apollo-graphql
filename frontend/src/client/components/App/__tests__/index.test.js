import React from 'react';
import { shallow } from 'enzyme';
import App from '../index';

describe('components: <App />', () => {
  test('should render correctly', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
