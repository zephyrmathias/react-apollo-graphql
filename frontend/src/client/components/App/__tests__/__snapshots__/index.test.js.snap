// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components: <App /> should render correctly 1`] = `
<div>
  <Header />
  <div>
    App
  </div>
  <AppRoutes />
</div>
`;
